### ISSUE TITLE
Setting to optionally make the "Read" function in the bookmarks section move the scroll position to the bottom of the thread

### ISSUE THREAD

**Issue by crisalis2** [K1rakishou/Kuroba-Experimental#790]
Right now in the bookmarks section there is a function to "Read" a thread, but that just makes it show 0 new posts in the bookmarks section only. If the user actually navigates to the thread, the unseen posts are still unseen and the scroll position has not changed since the last time the user was there. 

I understand the current behavior may have some benefit for some users, but I personally wish there was a setting to optionally make the "Read" funtion also move the scroll position to the bottom of the thread ("off-screen"), so that what the user achieves when using this function is not just that the thread shows 0 new posts in the bookmarks section only, but also that if he navigates to the thread he ends up directly in its bottom [or more exactly the position in the thread that was the bottom at the point in time he used the "Read" function] and there are no unseen posts (no side lide marking unseen posts, no horizontal line marking where unseen posts start, no "X new posts" mini notification) [unless some time has passed between the point in time the user used the "Read" function and the point in time he is navigating to thread, and in that lapse of time new posts have appeared].

TL;DR I just want "Read" to also scroll to bottom.
